packages/graphiql/src/new-components/themes/decorators.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
needs path rewrite?

Reply:
@acao Sorry not following what you're suggesting?

Reply:
decorators.js -> tsx. sorry just like to keep this stuff in order because its an OSS project

Reply:
Ah. I misunderstood what you were asking.